int.ts

```diff
@@ -443,9 +443,9 @@ module J2ME {
         frame.setParameter(kinds[i], index++, arguments[i]);
       }
       if (methodInfo.isSynchronized) {
-        var monitor = methodInfo.isStatic ? methodInfo.classInfo.getClassObject() : getMonitor(arguments[0]);
+        var monitor = methodInfo.isStatic ? methodInfo.classInfo.getClassObject()._address : arguments[0];
```

Comment:
Nit: I would rename this (and the other *monitor* variable below) to *monitorAddr*, since otherwise the `getMonitor(monitor)` calls are a bit confusing.